spec/colorizer.rb

```diff
@@ -20,7 +20,7 @@
       {rubymotion: "rocks!"},
       Time.now,
       Object.new,
-      UIView.new,
+      NSSet.new,
```

Comment:
I took the easy way -- It didn't seem worth creating different platform files...